project/dexter-vs/Dexter/PeerReview/Core/PeerReviewTaggerProvider.cs

```diff
@@ -22,6 +23,12 @@ public class PeerReviewTaggerProvider : ITaggerProvider
         [Import]
         public IClassifierAggregatorService AggregatorService;
 
+        [Import]
+        IDexterClient dexterClient;
```

Comment:
@KarolAntczak Thanks for your fix.

The import attribute is being used incorrectly. This causes the PeerReviewTaggerProvider to fail to execute properly.

In the case of the AggregatorService, the import attribute works normally because it is a service exported by VS Editor. However, because dexterClient and dexterInfoProvider are our own classes, we need to provide an environment to export them in order to use import attributes.